File: Backend/immersive_project_management_service/src/main/java/i5/las2peer/services/immersiveProjectManagementService/i5/las2peer/services/immersiveProjectManagementService/dataModel/gitHub/GitHubRepository.java
package i5.las2peer.services.immersiveProjectManagementService.i5.las2peer.services.immersiveProjectManagementService.dataModel.gitHub;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

/**
 * Created by bened on 20.07.2019.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class GitHubRepository {

    private int id;
    private String name;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: Frontend/VIAProMa/Assets/Scripts/DataModel/GitHub/GitHubRepository.cs
using i5.VIAProMa.UI.ListView.Core;

namespace i5.VIAProMa.DataModel.GitHub
{
    /// <summary>
    /// Represents a requirement in the Requirements Bazaar
    /// </summary>
    public class GitHubRepository
    {
        private int id;
        private string name;

        public int getId()
        {
            return id;
        }

        public string getName()
        {
            return name;
        }
    }
}